discovery/rendezvous.go

```diff
@@ -110,7 +110,7 @@ func (r *Rendezvous) register(topic string) error {
 
 // Register renews registration in the specified server.
 func (r *Rendezvous) Register(topic string, stop chan struct{}) error {
-	ticker := time.NewTicker(r.registrationPeriod)
+	ticker := time.NewTicker(r.registrationPeriod / 2)
```

Comment:
Why do we divide it by 2?

Reply:
seemed like a reasonable approach. the goal is to update before registration period will come to an end. so dividing by 2 almost always ensures that.

Reply:
i can use explicit configuration if current approach seems reasonable only to me

Reply:
yeah, it deserves at least an explicit comment with an explanation.